src/mgr/DaemonServer.cc

```diff
@@ -1998,6 +1998,16 @@ bool DaemonServer::handle_command(MCommand *m)
     }
     return true;
   } else {
+    if (!pgmap_ready) {
+      ss << "Warning: due to ceph-mgr restart, some PG states may not be up to date\n";
+    }
+    if (f) {
+       f->open_object_section("pgmap_global");
+       f->open_object_section("pgmap_ready");
+       f->dump_bool("pgmap_ready", pgmap_ready);
```

Comment:
i don't think we need to enclose "pgmap_ready" with an object. in json, it should be a boolean property of its enclosing object.

i don't really understand what "global" means.

and "pgmap_ready" is a little bit redundant in this context. "ready" would suffice.

Reply:
Yes, ”pgmap_ready” is bool type and does not need to be set to object, updated
other, "pgmap_global" changed to “pg_info” and  "pgmap_ready" changed to "pg_ready"

The purpose of preserving the pg prefix is to match “pg_stats”, In addition, these attributes belong to pg dump.

thank you